app/scripts/directives/accountBalanceTable.js

```diff
@@ -50,7 +50,9 @@ module.exports = function accountBalanceTable(
                 const _date = new Date();
                 const {
                     time: _time,
-                    amount
+                    amount: amount,
+					multiplier: multiplier,
```

Comment:
can you put the same indentation for the code? 2 or 4 spaces is ok, but everything should be in the indentation

Reply:
thanks, just made a style fix commit.